admin/client/App/screens/List/components/ItemsTable/ItemsTableRow.js

```diff
@@ -38,7 +38,7 @@ const ItemsRow = React.createClass({
 		});
 		// item fields
 		var cells = this.props.columns.map((col, i) => {
-			var ColumnType = Columns[col.type] || Columns.__unrecognised__;
+			var ColumnType = Columns[col.type] || Columns.text;
```

Comment:
@JedWatson how would you handle cases where a list does not have a name mapping?  In that case,  the ID, which doesn't not have a representative field shows.  That's what the `text` default is doing here ;=)